### PR TITLE
Isochrone : improve processing time

### PR DESCRIPTION
**Pull request by ZoeBrunet**
Remove useless check in merge_poly
Use symmetry to draw circles and build points each two degrees instread
of each one degree

### CODE REVIEW

File: source/routing/isochrone.cpp [hunk 1]
```diff
@@ -108,39 +108,48 @@ type::GeographicalCoord project_in_direction(const type::GeographicalCoord& cent
     return type::GeographicalCoord(lon_deg, lat_deg);
 }
 
+static type::GeographicalCoord build_sym (type::GeographicalCoord point,
```
Comment (kinnou02) [inline source/routing/isochrone.cpp]: const&

File: source/routing/isochrone.cpp [hunk 2]
```diff
@@ -108,39 +108,48 @@ type::GeographicalCoord project_in_direction(const type::GeographicalCoord& cent
...
+    for (unsigned int i = 0; i <= 90; i += 2) {
         points_out.push_back(project_in_direction(center, i, radius));
     }
+    for (unsigned int j = 90; j <= 270; j += 90) {
+        for (unsigned int k = 1; k <= 45; k++) {
+            type::GeographicalCoord point;
+            point = points_out[j / 2 - k];
+            type::GeographicalCoord point_sym;
+            point_sym = build_sym(point, center, j);
+            points_out.push_back(point_sym);
```
Comment (TeXitoi) [inline source/routing/isochrone.cpp]: you can do these 3 lines in one:

``` c++
points_out.push_back(build_sym(point, center, j));
```

File: source/routing/isochrone.cpp [hunk 3]
```diff
@@ -108,39 +108,48 @@ type::GeographicalCoord project_in_direction(const type::GeographicalCoord& cent
...
-               //We don't merge the polygons
-               multi_polys_merged.push_back(p);
-               log4cplus::Logger logger = log4cplus::Logger::getInstance("logger");
-               LOG4CPLUS_WARN(logger, "impossible to merge polygon: " << e.what());
-           }
-        } else {
-            multi_polys_merged.push_back(p);
+static type::MultiPolygon merge_poly(const type::MultiPolygon& multi_poly,
+                                     const type::Polygon& poly) {
+        type::MultiPolygon poly_union;
```
Comment (TeXitoi) [inline source/routing/isochrone.cpp]: bad indentation

**Comment by TeXitoi** [comment]
OK for me.

**Comment by xlqian** [comment]
ok for me

**Pull request merged**